config.js

```diff
@@ -36,9 +43,13 @@ System.config({
     "bootstrap": "github:twbs/bootstrap@3.3.7",
     "clean-css": "npm:clean-css@3.4.25",
     "core-js": "npm:core-js@1.2.7",
+    "css": "github:systemjs/plugin-css@0.1.33",
```

Comment:
Is (manual) update of the config.js sufficient (as different from `jspm install github:systemjs/plugin-css@0.1.33)?